src/components/diceManager.js

```diff
@@ -1,27 +1,82 @@
+import { wait } from "../utils";
+
+const FACE_ROTATION_TABLE = [
+  [3, 6, 4, 5],
+  [1, 6, 2, 5],
+  [4, 6, 3, 5],
+  [2, 6, 1, 5],
+];
+const PROCESS_DICE_DELAY = 4500;
+
+function mod(n, m) {
+  return (((n % m) + m) % m).toFixed(2);
+}
+
+function convertRadiansToNumRotation(phi) {
+  const half_pi = Math.PI / 2;
+  phi = phi.toFixed(2);
+  phi = mod(phi, 2 * Math.PI);
+  const num_rotation = Math.round(phi / half_pi);
+  return num_rotation % 4;
+}
+
 AFRAME.registerComponent("dice-manager", {
   schema: {
     total_dices: {
       type: "number",
       default: 0,
     },
+    dice_results: {
+      type: "string",
+      default: "",
+    },
   },
   init: function () {
+    const diceManager = this;
     const el = this.el;
-    this.el.sceneEl.addEventListener("throwDice", function (e) {
+    const sceneEl = this.el.sceneEl;
+    this.el.sceneEl.addEventListener("throwDice", async function (e) {
       const { num_dices } = e.detail;
       for (var i = 0; i < num_dices; i++) {
         const diceEl = document.createElement("a-entity");
         diceEl.setAttribute("class", "dice");
         diceEl.setAttribute("dice", {
           pos_x: Math.random() * 6 - 3,
-          pos_y: Math.random() * 3 + 1,
-          pos_z: Math.random() * 3 + -4,
+          pos_y: Math.random() * 3 + 4,
+          pos_z: Math.random() * 3 + -7.5,
```

Comment:
I made a change here. So dice are in the middle of the screen instead of being to the right.
SO make sure you pull b4 making changes